etc/testing/cleanup.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+// ClusterInfo holds information about a cluster.
+type ClusterInfo struct {
+	KopsBucket      string `json:"kops_bucket"`
+	PachydermBucket string `json:"pachyderm_bucket"`
+	Created         string `json:"created"`
+}
+
+// KopsBucket is the s3 bucket used by kops.
+const KopsBucket = "pachyderm-travis-state-store-v1"
+
+// MaxClusterTime is the maximimum time a cluster can be up.
+const MaxClusterTime = time.Hour * 4
+
+// HandleRequest handles the deletion of old clusters.
+func HandleRequest() (string, error) {
+	cmd := exec.Command("/bin/sh", "-c", "export PATH=$PATH:/var/task; kops --state=s3://"+KopsBucket+" get clusters | tail -n+2 | awk '{print $1}'")
```

Comment:
minor suggestion: I generally prefer to call bash explicitly, since I'm sometimes surprised by what `/bin/sh` gives me (`/bin/sh` isn't bash in either ubuntu (dash) or alpine (ash), for example)